pkg/cli/cli_test.go

```diff
@@ -2097,6 +2104,16 @@ func checkNodeStatus(t *testing.T, c cliTest, output string, start time.Time) {
 			t.Errorf("value for %s (%d) greater than max (%d)", tc.name, val, tc.maxval)
 		}
 	}
+
+	if nodeCtx.statusShowDecommission || nodeCtx.statusShowAll {
+		names := []string{"is_decommissioning", "is_draining"}
+		for i := range names {
+			idx := baseIdx + i
```

Comment:
nit: wouldn't it be cleaner to use the same `baseIdx++` pattern as before? Otherwise any code added below will have a wrong baseIdx.

Reply:
Done.